polygeist/tools/cgeist/Lib/utils.cc

```diff
@@ -11,25 +11,28 @@
 #include "utils.h"
 #include "clang-mlir.h"
 
+#include "clang/AST/Expr.h"
+
 #include "mlir/IR/Builders.h"
 #include "mlir/IR/BuiltinOps.h"
 #include "mlir/IR/Operation.h"
 #include "mlir/IR/OperationSupport.h"
 #include "mlir/IR/Value.h"
+#include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/Dialect/MemRef/IR/MemRef.h"
 
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/StringRef.h"
-
-#include "mlir/Dialect/Linalg/IR/Linalg.h"
-#include "mlir/Dialect/MemRef/IR/MemRef.h"
-
-#include "clang/AST/Expr.h"
+#include "llvm/Support/WithColor.h"
```

Comment:
We should be able to remove this header from some files now?

Reply:
not from this one because it is used in the `warning()` function

Reply:
I already removed the #include for that header in other places though.

Reply:
Yup, see them on the new commit, so resolved this conversation.